### ISSUE TITLE
Add oak woodcutting script

### ISSUE THREAD

**Issue by AmasiaNalbandian** [takelley1/OCVBot#10]
Hello there, I'm interested in the project here and I saw that there's some easy to-dos in the test. 

I would like to give this one a try: 

# TODO: Add oak woodcutting script that waits by an oak tree, clicks on
#   it when it appears, and empties inventory when full -- super simple.

**Comment by takelley1** [comment]
Sounds good!

I'm fine with a more straightforward implementation, but I think the reason I hadn't implemented this myself was because I wanted a generic woodcutting script that could work with any oak tree. Since each tree might be rotated slightly differently, I wasn't sure how reliable the image matching would be, as the confidence threshold would have to be much lower to register a match. 

I'd love to see what you come up with though!

**Comment by AmasiaNalbandian** [comment]
@takelley1  You're right - proving extremely difficult. Should we just target certain trees in locations?

**Comment by takelley1** [comment]
That's fine too. However, you might also want to look into [PyAutoGUI's pixel-matching functions](https://pyautogui.readthedocs.io/en/latest/screenshot.html#pixel-matching).

Instead of matching against an image of the tree, you could match against one of the many bright green pixels that make up the tree leaves. None of the functions in `vision.py` use this method, so it will take a little work to implement, but it's definitely doable.
